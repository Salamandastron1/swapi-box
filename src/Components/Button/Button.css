.buttons {
  grid-area: buttons;
  background: blue;
}