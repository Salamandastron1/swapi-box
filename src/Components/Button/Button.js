import React, { Component } from 'react';
import './Button.css';

class Button extends Component => {
  constructor() {
    super()
    this.state = {

    }
  }

  render() {
    return (
      <section className="button-container">
        <button 
          onClick={console.log('Favorites')} 
          className='favorites-btn button'>
          <i className="fab fa-trade-federation"></i>
            Favorites=0
        </button>
        <button 
          onClick={() => searchStarWarsApi(directory.people)}  
          className='people-btn button' >
          <i className="fab fa-jedi-order"></i>
            People
        </button>
        <button 
          onClick={() => searchStarWarsApi(directory.planets)}  
          className='planets-btn button'>
          <i className="fab fa-empire"></i>
            Planets
        </button>
        <button 
          onClick={() => searchStarWarsApi(directory.vehiclesu)}  
          className='vehicles-btn button'>
          <i className="fab fa-galactic-republic"></i>
            Vehicles
        </button>
      </section>
    )
  }
}

export default Button;