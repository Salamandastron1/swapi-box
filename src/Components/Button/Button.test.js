import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Button from './Button'

describe('Button', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Button />);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});