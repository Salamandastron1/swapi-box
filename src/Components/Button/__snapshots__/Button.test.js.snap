// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches the snapshot 1`] = `
<section
  className="button-container"
>
  <button
    className="null favorites-btn button"
    onClick={[Function]}
  >
    <i
      className="fab fa-trade-federation"
    />
    Favorites=0
  </button>
  <button
    className="null people-btn button"
    onClick={[Function]}
  >
    <i
      className="fab fa-jedi-order"
    />
    People
  </button>
  <button
    className="null planets-btn button"
    onClick={[Function]}
  >
    <i
      className="fab fa-empire"
    />
    Planets
  </button>
  <button
    className="null vehicles-btn button"
    onClick={[Function]}
  >
    <i
      className="fab fa-galactic-republic"
    />
    Vehicles
  </button>
</section>
`;
